app/controllers/account/downloads_controller.rb

```diff
@@ -0,0 +1,31 @@
+class Account::DownloadsController < ApplicationController
+  def show
```

Comment:
create action "download" in Companies controller and remove this

ALSO make sure I can download ONLY MY companies.